AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -79,53 +80,65 @@ The Alembic exporter can be customized by setting the properties on this compone
 
```

Comment:
In the table below, some sentences end with dots, some don't. Make them all the same